swagger.yaml

```diff
@@ -2725,6 +2725,8 @@ definitions:
         example: '{"ignition": {"version": "3.1.0"}, "storage": {"files": [{"path": "/tmp/example", "contents": {"source": "data:text/plain;base64,aGVscGltdHJhcHBlZGluYXN3YWdnZXJzcGVj"}}]}}'
       installer_args:
         type: string
+      machine_config_pool_name:
+        type: string
```

Comment:
if you limit the size in the DB shouldn't you limit the size in the api as well?